PC/winreg.c

```
@@ -1398,6 +1416,8 @@ winreg.OpenKey -> HKEY
     access: REGSAM(c_default='KEY_READ') = winreg.KEY_READ
         An integer that specifies an access mask that describes the desired
         security access for the key.  Default is KEY_READ.
+    options: int = 0
+        Can be one of the REG_OPTION_* constants.
```

We should probably document the interaction with `reserved` here. I'm not sure what the history is of `reserved` becoming a meaningful parameter, but it's meaningful now, so we should say that here (and in the real docs, `Doc\library\winreg.rst`). (I mean "here" as in a few lines above, but I can't comment on it because it hasn't been updated yet).

---

We could make all optional parameters keyword-only. If there are many optional parameters, passing positional arguments is error-prone.

Also, imagine the code that calls `CreateKeyEx(key, sub_key, options, access)` was executed in the old Python version. It will silently ignore the third argument.

---

Hi, I updated and changed the newly added `reversed` to keyword-only. Change all the optional parameters to keyword-only should be in another PR I think?